README.md

```diff
@@ -2,10 +2,29 @@
 
 1. Install Rust (https://www.rust-lang.org/tools/install)
 
+#### Versions
+this was tested on:
+
+| dependency    | version
+| ------------- |:-------------:|
+| Rust | 1.45.0 |
+| Cargo | 1.45.0 |
+| Actix-Web | 0.7.19 |
+| Sentry SDK | 0.19.0 |
+
+
 ## Run
 
 1. go into the directory that has the Cargo.toml file in it
 2. run "cargo run"
 3. go to URL http://localhost:3001/checkout
-4. go to URL http://localhost:3001/handled_new
+4. go to URL http://localhost:3001/handled
 5. go to URL http://localhost:3001/unhandled
+
+## Testing
```

Comment:
Testing section here, is for general tips on how to go about testing.

The sentry issue links, you can put to the Description portion of the PR

Reply:
so you can remove the 'Testing' section from this readme here

Reply:
@sentryadam0000345